styles/components/HackathonModal.module.scss

```diff
@@ -0,0 +1,81 @@
+// Import fonts
+@import url('https://fonts.cdnfonts.com/css/motorblock');
```

Comment:
why are we loading the fonts here and in the javascript? (If we decided to use the nextjs font then we should probably remove these). Otherwise why are we loading in both places?